lintro/tools/core/tool_installer.py

```
@@ -0,0 +1,415 @@
+
+        Returns:
+            True if installed >= minimum.
+        """
+        try:
+            inst = [int(x) for x in installed.split(".")[:3]]
+            mini = [int(x) for x in minimum.split(".")[:3]]
+            while len(inst) < 3:
+                inst.append(0)
+            while len(mini) < 3:
+                mini.append(0)
+            return inst >= mini
```

Ruff (0.15.7)

[warning] Consider moving this statement to an `else` block

(TRY300)